### ISSUE TITLE
Channels no longer appear

### ISSUE THREAD

**Issue by Malcolmest** [trakt/Plex-Trakt-Scrobbler#549]
Now that channels no longer appear is it possible to run the app entirely from within the plugin settings? I've been trying and it seems to have synced my movies but not my TV shows. Would love for this to be a functional plugin going into the "plugin-less" plex days.

**Comment by matthewknill** [comment]
I have the same question, Plaxt 2 doesn't seem to work in favour of Trakt's support for Plex integration (which costs money). I would love to be able to use this even it it means editing from the plugin directory. Having the ability to sync down from Trakt (rather than up) is another awesome advantage over webhooks.

**Comment by matthewknill** [comment]
Ahhh you go to settings and then scroll to the bottom and click on plugins. If installed correctly you should see it there. I have yet to see if it still works.

**Comment by polishboi** [comment]
@matthewknill where exactly?

![obraz](https://user-images.githubusercontent.com/48173986/74433522-d6746280-4e60-11ea-8bcb-f41012c72405.png)

**Comment by matthewknill** [comment]
Oh I think I know what you mean, yeah the option for manually running it seems to be missing. However, new plays will be synced if you click the settings for the plugin etc...
![image](https://user-images.githubusercontent.com/50228686/74433778-2da42280-4eb5-11ea-8602-ac0281d460b6.png)
Apologies for my misunderstanding

**Comment by polishboi** [comment]
@matthewknill no problem,
still tv shows not syncing for me just like OP, I'd love to make a Pull manually somehow :/

**Comment by pekke74** [comment]
> @matthewknill no problem,
> still tv shows not syncing for me just like OP, I'd love to make a Pull manually somehow :/
> 
I have the same problem. A tenth of my TV series are updated. Has anyone found a solution?

**Comment by agneevX** [comment]
Right now, Kitana is one of the best ways to manage plugins.

**Comment by Malcolmest** [comment]
Yeah I got trakt to sync tv shows using Kitana. It’s a lifesaving front end
for the old channels interface. You can use any plugin just like before
channels was pulled from Plex.
